plugins/demote.js

```diff
@@ -1,6 +1,7 @@
-let handler = async (m, { conn, args, usedPrefix }) => {
+let handler = async (m, { conn }) => {
   let users = m.mentionedJid
-  for (let user of users) conn.groupDemoteAdmin(m.chat, user).catch(console.log)
+  if (!users.length) m.reply("tag membernya!")
+  else conn.groupDemoteAdmin(m.chat, users)
```

Comment:
Rawan banned kalo nggk satu per satu

Reply:
kalau satu satu error bwang :

TypeError: participants.map is not a function
    at WAConnection.groupQuery (/home/lelouch/wabot-aq/node_modules/@adiwajshing/baileys/lib/WAConnection/8.Groups.js:151:41)
    at WAConnection.groupDemoteAdmin (/home/lelouch/wabot-aq/node_modules/@adiwajshing/baileys/lib/WAConnection/8.Groups.js:128:61)
    at WAConnection.handler (/home/lelouch/wabot-aq/plugins/demote.js:3:32)
    at WAConnection.handler (/home/lelouch/wabot-aq/handler.js:201:26)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)

Reply:
[user] kalo satu2

Reply:
owalah oke bwang